cms/private_media/storages.py

```diff
@@ -0,0 +1,52 @@
+import logging
+from typing import TYPE_CHECKING
+
+from botocore.exceptions import ClientError
+from django.core.files.storage import FileSystemStorage
+from storages.backends.s3 import S3Storage
+
+if TYPE_CHECKING:
+    from django.db.models.fields.files import FieldFile
+
+
+logger = logging.getLogger(__name__)
+
+
+class PrivacySettingS3Storage(S3Storage):
+    # pylint: disable=W0223
```

Comment:
had to look it up. how about
```suggestion
    # pylint: disable=abstract-method
```

Reply:
Addressed in [ab6a27d](https://github.com/ONSdigital/dis-wagtail/pull/46/commits/ab6a27dc13b51e1360124de6c8ea07d86dc40576)